src/soroban/parsers.ts

```diff
@@ -139,7 +139,18 @@ function parseSuccessful(
             : xdr.ScVal.scvVoid()
         };
       })[0]
-    })
+    }),
+
+    stateDiff: sim.stateDiff?.length ?? 0 > 0
```

Comment:
this doesn't appear compatible to the [updated rpc sim response model, which named this 'stateChanges'](https://github.com/stellar/soroban-rpc/pull/120/files#diff-c134d0a3385557a986aef0002a42e8c44b95b35b234bfb7074e503639a34281cR151) ? would it be easier to follow the same terminology in sdk as well, i.e. use `stateChanges` instead of `stateDiff` ?

Reply:
oh right good point !! I will change it to stateChanges